src/config.ts

```diff
@@ -22,4 +22,8 @@ export const config = {
 
   // log level - trace/debug/info/warn/error
   logLevel: process.env.DWN_SERVER_LOG_LEVEL || 'INFO',
+
+  subscriptionsEnabled:
+    { on: true, off: false }[process.env.SUBSCRIPTIONS] ?? true,
```

Comment:
please change the environment variable to `DWN_SUBSCRIPTIONS`. I realize not all environment variables currently follow that convention, but I would like to move in that direction.